db/migrate/20170501201633_add_canceled_to_order_detail.rb

```diff
@@ -0,0 +1,19 @@
+class AddCanceledToOrderDetail < ActiveRecord::Migration
+  def up
+    add_column :order_details, :canceled_at, :datetime
+    add_column :order_details, :canceled_by, :integer
+    add_column :order_details, :canceled_reason, :string
+
+    Reservation.find_each do |reservation|
+      reservation.order_detail.update_attributes(canceled_at: reservation.canceled_at,
+                                                 canceled_by: reservation.canceled_by,
+                                                 canceled_reason: reservation.canceled_reason)
```

Comment:
Should we add the remove columns to the reservations table? Totally cool leaving that be for now since it probably makes going up and down in the database easier and quicker while this is a WIP.